docs/spelling_wordlist.txt

```diff
@@ -5,3 +5,9 @@ scm
 Threadless
 threadless
 youtube
+socio
```

Comment:
Maybe add these to the document and not globally?

Reply:
Totally, I added these because CI was failing and making me nervous, haha.  Let me address them properly as a follow up PR.  I have a few bits more related to doc.  Will pull in as part of that.